src/qibo/models/parameter.py

```diff
@@ -0,0 +1,42 @@
+"""Model for combining trainable parameters and possible features into circuit parameters."""
+
+
+class Parameter:
```

Comment:
@MatteoRobbiati why do we have two `Parameter` classes?

Reply:
Oh, right, I forgot to remove it from there. Thanks.